readme/MacBuild.md

```diff
@@ -80,71 +198,34 @@ Follow these steps to resolve the issue:
    
 Steps are from this [source](https://stackoverflow.com/questions/17980759/xcode-select-active-developer-directory-error/17980786#17980786). A big thanks to tjmetha and Rob Bednark!
 
-### 6. Build sndfile
-**Recording and playing audio is not yet supported for MacOS.**
-````bash
-./build-sndfile.sh
-````
-
-### 7. Build adwaita icon theme
-````bash
-jhbuild build adwaita-icon-theme
-````
-
-### 8. Build GtkSourceView
-````bash
-jhbuild build gtksourceview3
-````
-
-## Build Xournal++ and package it as .app
-
-### Automated step
+#### Using a custom moduleset to work around dependency errors
 
-````bash
-./complete-build.sh $HOME/gtk
-````
+If you have MacPorts or Homebrew installed, the jhbuild packages may pick up on 
+packages outside of jhbuild during the build process. Consequently, this can
+result in build or configure time failures.
 
-### Manual steps
+To work around this problem, you can create a custom jhbuild moduleset and use
+that for your jhbuild packages.
 
-#### Build Xournal++
-````bash
-export PATH="$HOME/.local/bin:$HOME/gtk/inst/bin:$PATH"
+First, clone the gtk-osx repository:
+```sh
+git clone https://gitlab.gnome.org/GNOME/gtk-osx.git
+```
 
-cmake -DCMAKE_INSTALL_PREFIX:PATH=$HOME/gtk/inst ..
-make -j 4
-make install
-````
+Then add the following lines to your `~/.config/jhbuildrc-custom` to make
+jhbuild use the modulesets in the repository you just cloned:
+```python
+use_local_modulesets = True
+moduleset = "gtk-osx.modules"
 
-#### Build App
-````bash
-./build-app.sh $HOME/gtk
-````
+# change this to point to your own user directory
+modulesets_dir = "/Users/jhbuilduser/gtk-osx/modulesets-stable"
+```
```

Comment:
If Xournal++ is built for the Gtk Mac blob, `jhbuilduser` should be `xournal-dev`. Maybe this could be said at some place, since it might save some time for a core team member, who doesn't know that.

Reply:
Replaced with something that should work with any username.